src/m365/pp/commands/dataverse/dataverse-table-list.ts

```diff
@@ -24,7 +24,7 @@ class PpDataverseTableListCommand extends PowerPlatformCommand {
   }
 
   public defaultProperties(): string[] | undefined {
-    return ['SchemaName', 'EntitySetName', 'IsManaged'];
+    return ['SchemaName', 'EntitySetName', 'LogicalName', 'IsManaged'];
```

Comment:
This was not really in scope of the issue, but it think it might be useful to add it so I'll let it pass.